### PR TITLE
26137: context.is_active() and context.add_callback() do not work whe…

### PR DESCRIPTION
**Pull request by SameerMahajan-GSLab**
…n using grpc.aio

@nicolasnoble

### CODE REVIEW

File: src/python/grpcio/grpc/_cython/_cygrpc/aio/server.pyx.pxi [hunk 1]
```diff
@@ -203,6 +203,13 @@ cdef class _ServicerContext:
     def invocation_metadata(self):
         return self._rpc_state.invocation_metadata()
 
+    def is_active(self):
+        return self._rpc_state.status_code == StatusCode.ok
```
Comment (lidizheng) [inline src/python/grpcio/grpc/_cython/_cygrpc/aio/server.pyx.pxi:207]: This is probably wrong, we might want to check if it is `None` instead. If a status code is given, the RPC is finished.

File: src/python/grpcio/grpc/_cython/_cygrpc/aio/server.pyx.pxi [hunk 2]
```diff
@@ -203,6 +203,13 @@ cdef class _ServicerContext:
...
+
+    def add_callback(self, callback):
+        self._rpc_state.callbacks.append(callback)
```
Comment (lidizheng) [inline src/python/grpcio/grpc/_cython/_cygrpc/aio/server.pyx.pxi:210]: Maybe we also need to check if the callbacks are consumed properly?

**Review by lidizheng — CHANGES_REQUESTED** [review]
@SameerMahajan-GSLab Can you also add unit tests, and update the interface?

https://github.com/grpc/grpc/blob/5b035265cee380eea6c163ac08e4a0d62a4340df/src/python/grpcio/grpc/aio/_base_server.py#L134

**Comment by SameerMahajan-GSLab** [comment]
@lidizheng I have incorporated your review feedback. Can you please guide me with unit tests? How they are organized? Where should I add grpc aio test for these changes?

**Comment by SameerMahajan-GSLab** [comment]
I signed the CLA clicking on "Details" against EasyCLA - Missing CLA Authorization and also got the  email  stating that it is signed. Why is the check still showed as failing here?

**Comment by SameerMahajan-GSLab** [comment]
creating another pull request as the first two commits in this do not have git user name  set which is causing the check to  fail.

**Pull request closed without merging**